### DISCUSSION TITLE
Farming Simulator 22 ?

### DISCUSSION THREAD

**Discussion by Allread** [GameServerManagers/LinuxGSM#3686] [Ideas]
Hi :) 

Do you plan to release FS22 Dedicated server ?

**Comment by PlayMTL** [comment]
If im right, Farming Simulator Dedicated Servers are only availabe for Windows. We dont support "Wine" for such scenarios.
So I don't think FS is comming to LinuxGSM.

**Comment by gOOvER** [comment]
> Hi :)
> 
> Do you plan to release FS22 Dedicated server ?

There is no FS22 Dedicated Server. It's only the Game an you need to copy all by Hand. Only Dedicated Servers are on paying Hosters ;)

> If im right, Farming Simulator Dedicated Servers are only availabe for Windows. We dont support "Wine" for such scenarios. So I don't think FS is comming to LinuxGSM.

FS22 is not compatible to Wine
